apps/bsd/src/screens/advanced_options_screen.tsx

```diff
@@ -36,7 +43,10 @@ export function AdvancedOptionsScreen({
   markThresholds,
   electionDefinition,
 }: Props): JSX.Element {
-  const { lockMachine } = useContext(AppContext);
+  const { lockMachine, logger, currentUserSession } = useContext(AppContext);
+  const currentUserType = useMemo(() => currentUserSession?.type ?? 'unknown', [
```

Comment:
Same question here re: `useMemo`. I won't spam any more of the same question after this.